extensions-core/kafka-indexing-service/src/main/java/io/druid/indexing/kafka/supervisor/KafkaSupervisor.java

```diff
@@ -635,7 +649,7 @@ private void addDiscoveredTaskToPendingCompletionTaskGroups(
     }
 
     log.info("Creating new pending completion task group for discovered task [%s]", taskId);
-    TaskGroup newTaskGroup = new TaskGroup(startingPartitions);
+    TaskGroup newTaskGroup = new TaskGroup(startingPartitions, Optional.<DateTime>absent());
```

Comment:
This looks like the supervisor is going to assume that discovered tasks don't have min message times, is that right?